sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.institutions.controller.getApplicationDetails.e2e-spec.ts

```diff
@@ -0,0 +1,150 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  authorizeUserTokenForLocation,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAuthRelatedEntities,
+  getInstitutionToken,
+  InstitutionTokenTypes,
+} from "../../../testHelpers";
+import {
+  createFakeInstitutionLocation,
+  saveFakeStudent,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { Institution, InstitutionLocation } from "@sims/sims-db";
+
+describe("ApplicationInstitutionsController(e2e)-getApplicationDetails", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let collegeF: Institution;
+  let collegeFLocation: InstitutionLocation;
+  let collegeCLocation: InstitutionLocation;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    // College F.
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      appDataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation(collegeF);
+    await authorizeUserTokenForLocation(
+      appDataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+    // College C.
+    const { institution: collegeC } = await getAuthRelatedEntities(
+      appDataSource,
+      InstitutionTokenTypes.CollegeCUser,
+    );
+    collegeCLocation = createFakeInstitutionLocation(collegeC);
+    await authorizeUserTokenForLocation(
+      appDataSource,
+      InstitutionTokenTypes.CollegeCUser,
+      collegeCLocation,
+    );
+  });
+
+  it(
+    "Should get the student application details when student has a submitted application " +
+      "for the institution (application with location id saved).",
```

Comment:
I gave the same comment on the previous PR following the same, I would prefer not to have the explanation "(application with location id saved)".

Reply:
Thanks for removing the `(application with location id saved)`.
Do we really need to break the line to have the `"for the institution."` in the second part?